setup.py

```diff
@@ -21,12 +21,15 @@
 ### \file
 ### This file builds the dppl and dppl.ocldrv extension modules.
 ##===----------------------------------------------------------------------===##
-import sys
-import numpy as np
 import os
+import sys
 import versioneer
-from setuptools import setup, Extension
+
 from Cython.Build import cythonize
+from setuptools import setup, Extension
```

Comment:
`setuptools` should be imported before Cython.
In https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html#configuring-the-c-build:
> Note that when using setuptools, you should import it before Cython, otherwise, both might disagree about the class to use here.

Reply:
Good catch. I will fix it.

Reply:
Fixed.